.buildkite/bk.integration.pipeline.yml

```diff
@@ -157,6 +157,21 @@ steps:
           machineType: "n1-standard-8"
           image: "family/platform-ingest-elastic-agent-rhel-8"
 
+  - group: "FIPS Verification test"
+    key: integration-tests-fips-verify
+    steps:
+      - label: "x86_64:fips"
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-x86-64-fips' --build ${BUILDKITE_TRIGGERED_FROM_BUILD_ID}
+          .buildkite/scripts/steps/integration_tests_tf.sh fips false
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
```

Comment:
do we actually need a VM with 30Gi of ram? If yes, I'd still recommend to switch to `n2-standard-8`, n1 is really an old relic that can end up using very old (and slow) hardware.

Reply:
I just copied from the template above